### DISCUSSION TITLE
Add more key binds to Mod Tap

### DISCUSSION THREAD

**Discussion by franman** [WootingKb/wootility-issues#193] [Ideas]
---
name: Feature request
about: More key slots are needed to add
title: 'Add more binds to Mod Tap'
labels: enhancement
assignees: 'Simon' & 'Tony'

---

**Is your feature request related to a problem? Please describe.**
Sometimes there are too many buttons to press to perform a simple move.  Take for example, in Doom Eternal, if I want to dodge to the left I have to press A and another key together.  Mod Tap is on the right track but it is limited only to a single key bind for tap and another for hold.

**Describe the solution you'd like**
Just like DKS, there are multiple key slots (4 to be exact) why not do the same for Mod Tap (for both Hold and Tap)? If possible, why not let the user specify the number of open key slots for Hold and Tap.

**Describe alternatives or workarounds you've considered**
For Doom Eternal I just press the middle mouse button for dodging while pressing simultaneously W,A,S or D to specify the direction of dodge

**Additional context**
N/A

### Additional information

**Platform information**
Windows 10

**Wootility and Firmware version**
Wootility v4.4.13
Firmware v2.6.3

**Keyboard Type**
Wooting 60HE ARM

**Comment by CaribouFrequency** [comment]
A further example I have been thus far unable to implement is binding CTRL + BACKSPACE to the backspace key, either through DKS or mod tap. It would be nice to use this feature to implement inconvenient key combinations with a single button press.
